packages/driver/src/cy/testConfigOverrides.ts

```diff
@@ -167,6 +167,18 @@ export class TestConfigOverride {
       unverifiedTestConfig: [],
     }
 
+    // TODO: remove when experimental overriding is supported
+    const experimentalRetryCfgKeys = [
+      'experimentalStrategy', 'experimentalOptions',
+    ]
+
+    const experimentalRetryConfigAttempted = Object.keys(resolvedTestConfig.unverifiedTestConfig?.retries || {})
+    .some((v) => experimentalRetryCfgKeys.includes(v))
+
+    if (experimentalRetryConfigAttempted) {
+      throw new Error('Cannot set experimental retries on individual tests')
```

Comment:
This error message should probably be in [error_messages.ts](https://github.com/cypress-io/cypress/blob/04a1156ccd6093441c17ea04c98570ef8470a239/packages/driver/src/cypress/error_messages.ts#L948) with a `docsUrl` and use `$errUtils.throwErrByPath`. It would also benefit from being a little more specific, like if they try to use `experimentalStrategy`, we say "`experimentalStrategy` cannot be configured on an individual test. It can only be set globally." and same for `experimentalOptions`. Also, is it possible to set one of the options on `describe` or `context`? If not, it should say something like "cannot be configure per-suite or per-test".